Magento_Sales/templates/order/items.phtml

```diff
@@ -72,17 +72,17 @@
                                    <?= /* @escapeNotVerified */ __('Gift Message') ?>
                                 </a>
 
-                                <?php 
+                                <?php
                                     $giftMessage       = $this->helper('Magento\GiftMessage\Helper\Message')
-                                                              ->getGiftMessageForEntity($item); 
+                                                              ->getGiftMessageForEntity($item);
```

Comment:
new var for $this->helper('Magento\GiftMessage\Helper\Message') will make it one line

Reply:
replaced